backend/apps/cache/services/cache_manager.py

```
@@ -423,10 +427,12 @@ def invalidate_cache_on_delete(sender, instance, **kwargs):
 try:
     manager = CacheManager()
     manager.invalidate_dependencies(instance)
-except Exception:
-    # Silently ignore cache invalidation errors
-    pass
-        pass
+except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)